src/client/components/SessionTimer/index.js

```diff
@@ -62,7 +64,6 @@ function SessionTimer(props) {
     clear();
     warningTimerId = setTimeout(() => {
       if (sessionStorage.getItem(AUTH_STRINGS.authToken)) {
-        setNumberOfMinutes(TIMEOUT_DISPLAY_TIME_IN_MINUTES);
         setShowWarningModal(true);
```

Comment:
Ooooh, I see what's happening. When I asked you to flip it, it breaks the "don't reset if the modal is showing" logic so we have to show the modal first.

Ok, I think we can flip the order again now that you're setting the value on page load. Otherwise, if you dismiss the dialog and timeout again, it's not going to be set at 5min any more.

Which is to say, add back in `setNumberOfMinutes(TIMEOUT_DISPLAY_TIME_IN_MINUTES);` after `setShowWarningModal(true);`.

Reply:
Ahh you're right! Ugh!

Reply:
Thanks for catching that!